src/v5_hal/firmware/include/nodes/LiftNode.h

```diff
@@ -27,11 +28,11 @@ class LiftNode : public ILiftNode {
     void setLiftVelocity(int velocity);

```

Comment:
We should change the type to int - I think I implemented it incorrectly originally, but we should make this a float